### PR TITLE
Added just announced gpt-4o and gpt-4o-2024-05-13

### PR DESCRIPTION
**Pull request by ventz**
## Title

Added the just announced gpt-4o and gpt-4o-2024-05-13: https://platform.openai.com/docs/models/gpt-4o

## Relevant issues

Added just announced GPT4-o model:
https://platform.openai.com/docs/models/gpt-4o

## Type

🆕 New Feature

## Changes

Added:
gpt-4o
and
gpt-4o-2024-05-13

### CODE REVIEW

File: litellm/model_prices_and_context_window_backup.json [hunk 1]
```diff
@@ -9,6 +9,26 @@
...
+        "max_tokens": 4096, 
+        "max_input_tokens": 128000,
+        "max_output_tokens": 4096, 
+        "input_cost_per_token": 0.000015,
+        "output_cost_per_token": 0.00003,
+        "litellm_provider": "openai",
+        "mode": "chat",
+        "supports_function_calling": true
+    },
+    "gpt-4-turbo-2024-04-09": {
```
Comment (orby-fabian) [inline litellm/model_prices_and_context_window_backup.json:22]: incorrect name?

**Comment by ghost** [comment]
Closing in favor of #3613

File: litellm/model_prices_and_context_window_backup.json (hunk 1 above)
Comment (ventz) [reply litellm/model_prices_and_context_window_backup.json:22]: Seems correct per OpenAI model list. 

Seems gpt-4o is a gpt-4 variant, same as turbo.

**Comment by ishaan-jaff** [comment]
hi @ventz  can we hop on a call sometime this week. I'd love to learn how we can improve litellm for you. What's the best email to send an invite to ?

I DM'd on Linkedin if that's better
If it's easier here's a link to my cal https://calendly.com/d/4mp-gd3-k5k/litellm-1-1-onboarding-chat

**Pull request closed without merging**